parameters.yaml

```diff
@@ -6,3 +6,12 @@ parameters:
     required: true
     type: string
     description: Cluster ID
+
+  OrganizationIdPathParameter:
+    name: organization_id
+    in: path
+    required: true
+    type: string
+    description: |
+        An ID for the organization to be created is the only required parameter.
+        This ID can only contain alpha-numeric characters, hyphens, underscores, and dots.
```

Comment:
But since this is supposed to be about the spec. I am ok with writing what we _want_ it to be, and then fixing this in the correct microservice